python/test/unit/language/test_issue_3013.py

```diff
@@ -0,0 +1,27 @@
+import triton
+import triton.language as tl
+import torch
+import pytest
+
+
+@triton.jit
+def kernel(in_ptr, in_stride0, in_stride1,
+           out_ptr,
+           SIZE0: tl.constexpr, SIZE1: tl.constexpr):
+    offset0 = tl.arange(0, SIZE0) * in_stride0
+    offset1 = tl.arange(0, SIZE1) * in_stride1
+    block = tl.load(in_ptr + offset0[:, None] + offset1[None, :])
+    block_min = tl.min(block, axis=1)[:, None]
+    max_val = tl.max(block_min, axis=0)
+    tl.store(out_ptr + tl.arange(0, 1), max_val)
+
+
+def test_issue_3033():
```

Comment:
This doesn't look like a good test name.

Can you integrate it into test_core.py?

Reply:
done

Reply:
I'm thinking that it might be just one shape or layout missing in `test_reduce`. Can you take a look? thanks